<h2>Why is this an issue?</h2>
<p>Shared naming conventions allow teams to collaborate efficiently. This rule checks that all function names match a provided regular expression.</p>
<h3>Noncompliant code example</h3>
<p>With default provided regular expression:</p>
<pre>
def DoSomething( ) : Unit = { // Noncompliant
   // ...
}
</pre>
<h3>Compliant solution</h3>
<pre>
def doSomething( ) : Unit = {
   // ...
}
</pre>

