<h2>Why is this an issue?</h2>
<p>Most of the time a block of code is empty when a piece of code is really missing. So such empty block must be either filled or removed.</p>
<h3>Noncompliant code example</h3>
<pre>
def compute(a, b)
  sum = a + b
  if  sum &gt; 0 # Noncompliant; empty on purpose or missing piece of code?
  end
  puts "Result: #{sum}"
end
</pre>
<h3>Compliant solution</h3>
<pre>
def compute(a, b)
  sum = a + b
  if  sum &gt; 0
    puts "Positive result"
  end
  puts "Result: #{sum}"
end
</pre>
<h3>Exceptions</h3>
<p>When a block contains a comment, this block is not considered to be empty.</p>
<p><code>while</code> and <code>unless</code> loops are also exception to the rule.</p>
<pre>
while @order.process_next; end # Compliant
</pre>

